Sources/AirRobeWidget/Views/OptInView/AirRobeOptInView.swift

```diff
@@ -13,8 +13,8 @@ import Combine
 final class AirRobeOptInView: UIView, NibLoadable {
     @IBOutlet weak var widgetStackView: UIStackView!
     @IBOutlet weak var mainContainerView: UIView!
-    @IBOutlet weak var mainContainerExpandButton: UIButton!
-    @IBOutlet weak var margin: UIView!
+    @IBOutlet weak var topContentContainer: UIStackView!
```

Comment:
Good to use a UIStackView; it's a good practice when usable space is unknown or variable (different screen sizes)

However, I wonder if perhaps the new view hierarchy is partly responsible for the layout constraint warnings? Might be worth investingating.